src/sentry/models/sentryappwebhookerror.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import EncryptedJsonField, FlexibleForeignKey, Model
+
+
+class SentryAppWebhookError(Model):
+    __core__ = False
+
+    date_added = models.DateTimeField(default=timezone.now)
+
+    sentry_app = FlexibleForeignKey("sentry.SentryApp", related_name="webhook_errors")
+
+    organization = FlexibleForeignKey(
+        "sentry.Organization", related_name="sentry_app_webhook_errors"
+    )
+
+    # If there is a related Sentry error, store it here
+    error_id = models.CharField(max_length=64, null=True)
+
+    request_body = EncryptedJsonField()
```

Comment:
We'll have to make sure we do `json.dumps(request.body)` when we save this field. It could end up being a plain string if the response is HTML.

Reply:
@mnoble But this is the request body, not response. The request is always going to be JSON as that's what we send on the webhook.

Reply:
Yeah, the request_body should already be JSON since we're sending it